contrib/bradfitz/gomemcache/memcache/memcache.go

```diff
@@ -80,6 +80,11 @@ func (c *Client) startSpan(resourceName string) ddtrace.Span {
 func (c *Client) Add(item *memcache.Item) error {
 	span := c.startSpan("Add")
 	err := c.Client.Add(item)
+	span.SetTag("item.key", item.Key)
+	if c.cfg.withValueTags {
+		span.SetTag("item.value", item.Value)
+		span.SetTag("item.expiration", item.Expiration)
```

Comment:
Why is this also disabled when values are? Is this also considered sensitive information?

Reply:
Do we want to have this always or only when values are also there?

Reply:
I wondered it should be included or not. I just excluded all information except key.

But If you think it should be included, i’m ok too.

Reply:
I think so too, but I wasn't sure. Let's include it then. Only value remains toggleable.

Reply:
Yap. Done!